### PR TITLE
Added leetcode solution for 136 Single Number

### LINKED ISSUE #816: Add Leetcode solution for 136  Single Number
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

Add Leetcode solution for 136  Single Number

### Suggested Change

User can find solution

### Rationale

_No response_

### Urgency

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Hemav009**
## Related Issue

Fix #816 

## Description

Added leet code solution for 136 Single Number

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md [hunk 1]
```diff
@@ -1,89 +1,104 @@
 ---
-id: letter-combinations-of-a-phone-number
+id: Letter Combinations of a Phone Number
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md:2]: replace `id: Letter Combinations of a Phone Number` to `id: letter-combinations-of-a-phone-number`

File: dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md [hunk 2]
```diff
@@ -1,89 +1,104 @@
...
 title: Letter Combinations of a Phone Number (LeetCode)
-sidebar_label: 0017-Letter Combinations of a Phone Number
+sidebar_label: 0017-Letter-Combinations-of-a-Phone-Number
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md:4]: replace `sidebar_label: 0017-Letter-Combinations-of-a-Phone-Number` to `sidebar_label: 0017-Letter Combinations of a Phone Number`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by Hemav009** [comment]
@Ajay-Dhangar  Updated

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**